docs/pages/build/building-on-ci.md

```diff
@@ -156,26 +156,17 @@ version: 2.1
 executors:
   default:
     docker:
-      - image: circleci/node:16
+      - image: cimg/node:lts
     working_directory: ~/my-app
 
-commands:
-  attach_project:
```

Comment:
This command isn't adding any values to the configuration sample as we're never persisting a workspace anyway.

Removing this to avoid confusion and errors.